attribute/attribute_test.go

```diff
@@ -55,5 +55,5 @@ func TestAttributeKeysUniquePointers(t *testing.T) {
 	// were inadvertently defined as an empty struct, then
 	// NewKey would always return the same pointer. This
 	// guards against such a mistake.)
-	assert.NotSame(t, NewKey[string](), NewKey[string]())
+	assert.NotSame(t, NewKey[string](), NewKey[string]()) //nolint:testifylint
```

Comment:
What lint issue is this suppressing? Maybe add a comment about this so it's more clear?
```suggestion
	//nolint:testifylint // explain here why we are ignoring this rule...
	assert.NotSame(t, NewKey[string](), NewKey[string]())
```